docs/uml-class-diagrams/middleware01/test/MiddlewareGisProcessorImplementationTests/MiddlewareGisProcessorImplementationTests.puml

```diff
@@ -1,9 +1,11 @@
 @startuml MiddlewareGisProcessorImplementationTests
 
 package edu.oakland.test.middleware01 {
-    class MiddlewareGisProcessorImplementationTests {
-
-    }
+  class MiddlewareGisProcessorImplementationTests {
+    + void isGpsSignalStrengthNull()
+    + void checkValueGpsSignalStrength()
```

Comment:
Just double checking: This class is entirely a passthrough, correct? If so, then we can probably do something like this:
```suggestion
    + void isGpsSignalStrengthNull()
    + void checkValueGpsSignalStrength()
```
If this class is not a pass through, and there is logic there, you'll need to cover each branch, so this might look something like:
```suggestion
    + void evaluateGpsSignalStrengthRejectsIfTooLow()
    + void evaluateGpsSignalStrengthAcceptsIfValid()
    + void checkValueGpsSignalStrengthDisconnected()
    + void checkValueGpsSignalStrengthConnectedAndStop()
```
Or it might be some mix of those if some are pass through and some have logic. Either way though, these are much better, and now it's just about getting the coverage right, rather than what they are doing.

Reply:
By my understanding it's a passthrough
![FoxitReader_2020-11-27_19-26-29](https://user-images.githubusercontent.com/59949033/100490078-76486780-30e6-11eb-89bf-86a4d9882224.png)

Reply:
Copy that! Just know that it could change because there is an if statement I think on the UML Activities Diagram, but the goal of this is just to give you a framework and nail down the edge cases; code coverage will reveal any missed paths if they come up.